src/sanitiser.js

```diff
@@ -1,46 +1,82 @@
-import { globalConfig, testConfig } from './defaultConfig';
-
-const hasProperty = (obj, property) =>
-  Object.prototype.hasOwnProperty.call(obj, property);
-
-const sanitiseTestConfiguration = (conf) => {
-  const configuration = conf || {};
-  configuration.resolution = hasProperty(configuration, 'resolution')
-    ? configuration.resolution
-    : testConfig.resolution;
-  configuration.steps = hasProperty(configuration, 'steps')
-    ? configuration.steps
-    : testConfig.steps;
-  configuration.name = hasProperty(configuration, 'name')
-    ? configuration.name
-    : testConfig.name;
+import check from 'check-types';
+import type from 'type-detect';
+import logger from './utils/logger';
+import { globalConfig, testConfig } from './config/config';
+
+const error = (name, wrongType, correctType) =>
```

Comment:
rename this to logError to make it clearer what it does

Reply:
Done